NICE.Identity.Management/Views/PermissionDenied.cshtml

```diff
@@ -5,7 +5,7 @@
 <!DOCTYPE html>
 <html>
 	<head>
-		<link rel="stylesheet" href="https://www.nice.org.uk/consultations/styles/main.css" />
+		<link rel="stylesheet" href="nice-design-system.css" />
```

Comment:
Should this be pointing to the CDN?

Reply:
I've pointed it to a local version of the installed design system so that it's consistent with the rest of the app. Think we were trying to avoid linking to external files for one of the previous pen test assessments.